cmd/scw/testdata/test-all-usage-container-deploy-usage.golden

```diff
@@ -0,0 +1,22 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Automatically build and deploy a container.
+
+USAGE:
+  scw container deploy [arg=value ...]
```

Comment:
scw container container deploy we have to stay consistent

Reply:
This command already exists, it's an API endpoint